src/toolbar/toolbar-group.jsx

```diff
@@ -19,17 +18,20 @@ function getStyles(props, state) {
 
   const styles = {
     root: {
-      float,
       position: 'relative',
       marginLeft: firstChild ? -marginHorizontal : undefined,
       marginRight: lastChild ? -marginHorizontal : undefined,
+      display: 'flex',
+      justifyContent: 'space-between',
+      flexWrap: 'wrap',
```

Comment:
@tintin1343 When I look through the spec, I can't find any examples of toolbars having their icons wrap onto new rows. (It's not really a 'bar' if we are automatically wrapping children onto new rows)

IMO it is the user's responsibility to make sure they are setting up their layout so there are not too many items at lower resolutions.

When we extend the height of toolbars, it should be for specific purposes like:
![image](https://cloud.githubusercontent.com/assets/4420103/13465810/32b61fb6-e065-11e5-80dd-97904814ba7e.png)